fsharp-backend/tests/Tests/DvalRepr.Tests.fs

```diff
@@ -38,7 +38,7 @@ let testDvalRoundtrippableRoundtrips =
     "special roundtrippable dvals roundtrip"
     FuzzTests.OCamlInterop.Roundtrippable.roundtrip
     [ RT.DObj(
-        Map.ofList [ ("", RT.DFloat 1.797693135e+308)
+        Map.ofList [ ("", RT.DFloat 1.797693135e+308) // WHATISTHIS why does it work without an attribute name
```

Comment:
I've seen this in a few places. Why do DObjs with "empty" attribute/property names _work_?

Reply:
This is because a DObj is both a dictionary and a record (kinda like JS objects or php dicts I guess). Dictionaries can have `""` as a valid key.

Detangling this problem is for https://github.com/darklang/dark/issues/3406

Reply:
Got it!